packages/react-mutation-mapper/src/component/track/TrackItem.tsx

```diff
@@ -27,6 +27,7 @@ export type TrackItemSpec = {
     label?: string;
     color?: string;
     tooltip?: JSX.Element;
+    rectHeight?: number; // set rectangle height (only works if itemType is RECTANGLE)
```

Comment:
Can we try using `dim2` instead of introducing a new height prop?

https://github.com/cBioPortal/cbioportal-frontend/blob/3675a07a438ad31562b5945e22f71d1be8e7de55/packages/react-mutation-mapper/src/component/track/TrackItem.tsx#L11

Looks like it is hardcoded to `22` in `Track.tsx`. So maybe we can set its default value to `22`, and set it to `15` for the new uniprot track?

https://github.com/cBioPortal/cbioportal-frontend/blob/639003804a8d68bea8f2fd1a0e53e8a28dff7a73/packages/react-mutation-mapper/src/component/track/Track.tsx#L220